docs/userguide.md

```diff
@@ -128,76 +143,136 @@ Status:
 
 Follow these instructions for running an example to verify your deployment is
 working. The example will create a test namespace with a
-`federatednamespaceplacement` resource as well as federated template,
-override, and placement resources for the following k8s resources: `configmap`,
-`secret`, `deployment`, `service` and `serviceaccount`. It will then show how
-to update the `federatednamespaceplacement` resource to move resources.
+`federatednamespaceplacement` resource as well as federated template, override,
+and placement resources for the below k8s resources. It will then show how to
+update the `federatednamespaceplacement` resource to move resources. So let's
+set a variable with all the resource types for convenience:
+
+```bash
+RESOURCE_TYPES="deployment ingress job secret service"
+```
 
 ### Create the Test Namespace
 
 First create the `test-namespace` for the test resources:
 
 ```bash
-kubectl apply -f example/sample1/federatednamespace-template.yaml \
-    -f example/sample1/federatednamespace-placement.yaml
+kubectl apply -f example/propagation/federatednamespace-template.yaml
+```
+
+### Create the Test Namespace Placement
+
+Now we create the `federatednamespaceplacement` resource for `test-namespace`.
+First replace the `null` string with the `HOST_CLUSTER` so that the federated
+controller does not prematurely delete the namespace, and all its contained
```

Comment:
Please make it clear that deletion of the namespace will be from member clusters.  The way this reads suggests the host cluster namespace could be removed, and it risks unnecessarily alarming the user as to the potential of  namespace placement to do harm.